src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/LegacyPbaResponse.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.professionalapi.controller.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+public class LegacyPbaResponse {
+
+    @JsonProperty
+
+    @SuppressWarnings({"checkstyle:MemberName"})

```

Comment:
We should only do this when absolutely necessary. Here you could easily have written a junit test wit a little bit of reflection to check this field

Reply:
Will have this added to RDCC-210 to remove this and add proper test-coverage instead.

Reply:
This not for JUnit ..its violating our checkstyle since legacy endpoint need response in the payment_accounts so to overcome this added the above rule.